ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml

```diff
@@ -54,6 +54,12 @@ tests:
     clone: true
     from: test-bin
   optional: true
+- as: kube-lint
+  commands: |
+    make kube-lint
+  container:
+    from: src
+  optional: true
```

Comment:
Should we make this required? If there is something wrong here it might not deploy correctly.

Reply:
I rather have new jobs (as we did with `helm-lint`) optional for few days to make sure they are stable before making them mandatory.

I will make all optional jobs mandatory in 2 or 3 days.

Reply:
I have added a task for it so we make sure I don't forget https://issues.redhat.com/browse/MGMT-8575.